actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -209,8 +209,12 @@ private void validateUserOrganisations(Request actorMessage, boolean isPrivate)
             userOrg.put(JsonKey.HASH_TAG_ID, organisation.getHashTagId());
             if (userOrg.get(JsonKey.ROLES) != null) {
               RoleService.validateRoles((List<String>) userOrg.get(JsonKey.ROLES));
+              if (!((List<String>) userOrg.get(JsonKey.ROLES))
```

Comment:
reduce complexity of code - do not fetch same variable from map again and again.